Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2344,6 +2344,114 @@ public Expression resolveIdentifier(String identifier, boolean mustResolve) {
         return null;
     }
 
+    public QueryLetRef getQueryLetRef(LetClause let) {
+        QueryLetRef result = of.createQueryLetRef().withName(let.getIdentifier());
```

Comment:
suggestion: most of the XYZ ref factory functions could be reused directly inline in the library builder such as here:
https://github.com/cqframework/clinical_quality_language/blob/b1b2854f80fdb5b0d342cccc92bb4a51058223aa/Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java#L2232

Could also take that as a second PR to avoid mixing refactoring into this one.

Reply:
I deleted this method since I no longer need to deal with Expressions.